src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapSessionOptions.Linux.cs

```diff
@@ -57,8 +57,9 @@ public bool SecureSocketLayer
 
         public int ProtocolVersion
         {
-            get => GetPtrValueHelper(LdapOption.LDAP_OPT_VERSION).ToInt32();
-            set => SetPtrValueHelper(LdapOption.LDAP_OPT_VERSION, new IntPtr(value));
+            get => GetIntValueHelper(LdapOption.LDAP_OPT_VERSION);
+
```

Comment:
Nit: no spacing between property accessors.

Reply:
> Nit: no spacing between property accessors.

removed the extra spaces